cpp/include/raft/neighbors/cagra.cuh

```diff
@@ -57,7 +67,8 @@ namespace raft::neighbors::cagra {
  *   ivf_pq::search_params search_params
  *   auto knn_graph      = raft::make_host_matrix<IdxT, IdxT>(dataset.extent(0), 128);
  *   // create knn graph
- *   cagra::build_knn_graph(res, dataset, knn_graph.view(), 2, build_params, search_params);
+ *   cagra::build_knn_graph(res, dataset, knn_graph.view(),
+ *   raft::distance::DistanceType::L2Expanded, 2, build_params, search_params);
```

Comment:
Please initialize build params in line 66 using `get_default_ivf_pq_build_params`
```suggestion
 *   cagra::build_knn_graph(res, dataset, knn_graph.view(), 2, build_params, search_params);
```